com.unity.visualeffectgraph/Editor/GraphView/Elements/Controllers/VFXDataAnchorController.cs

```diff
@@ -123,6 +123,10 @@ protected override void ModelChanged(UnityEngine.Object obj)
             UpdateInfos();
             Profiler.EndSample();
 
+            if (m_GizmoContext != null)
+            {
+                m_GizmoContext.Unprepare();
+            }
```

Comment:
It should be `RefreshGizmo` here, however, this function is called everytime a value is changed...
Do you have any update for this ? I recall we discussed about caching the expression hash to check if it changed, not sure it's a viable solution 🤷

Reply:
Yes, works as expected 👍